Sources/Clappr_iOS/Classes/Plugin/Core/Drawer/DrawerPlugin.swift

```diff
@@ -29,6 +29,7 @@ open class DrawerPlugin: OverlayPlugin {
         didSet {
             let event: Event = isClosed ? .didHideDrawerPlugin : .didShowDrawerPlugin
             core?.trigger(event)
+            isClosed ? hide() : show()
```

Comment:
I think that we can group this logic, once the rule is verify if plugind isClosed. What do you think?